ember/app/controllers/contact.js

```diff
@@ -49,13 +49,13 @@ export default Ember.ObjectController.extend({
       });
 
       this.set("modalClosed", false);
-      Ember.$("body").addClass("modal-backing");
+      Ember.$(".main-container").addClass("modal-backing");
```

Comment:
Yeah, this is actually a _bad practice_ but I'm not quite sure how to toggle a class on the body without it :scream: